### ISSUE TITLE
Value 'M.shape' is unsubscriptable

### ISSUE THREAD

**Issue by sursu** [PyCQA/pylint#3228]
This: https://github.com/microsoft/vscode-python/issues/8300

**Comment by PCManticore** [comment]
Cannot reproduce this with:
```
pylint 2.5.0-dev1
astroid 2.4.0
Python 3.6.5 (default, May  7 2018, 10:07:07)
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.42)]
```
Also please follow the issue template from now on, just a link to an external issue tracker is not a proper bug report.

**Comment by sursu** [comment]
If I am checking the `pylint` version correctly:
```python
import pylint
pylint.__versions
```
Then my version is 2.4.2. And, at least on Ubuntu 18.04.3, the issue is still there.
